src/js/builtins/DecompressionStream.ts

```diff
@@ -0,0 +1,33 @@
+export function initializeDecompressionStream(this, format) {
+  const zlib = require("node:zlib");
+  const stream = require("node:stream");
+
+  const builders = {
+    "deflate": zlib.createInflate,
+    "deflate-raw": zlib.createInflateRaw,
+    "gzip": zlib.createGunzip,
+    "brotli": zlib.createBrotliDecompress,
+    "zstd": zlib.createZstdDecompress,
+  };
+
+  if (!(format in builders))
+    throw $ERR_INVALID_ARG_VALUE("format", format, "must be one of: " + Object.keys(builders).join(", "));
```

Comment:
ngl i wonder the perf of this, but it is a really cold path so probably doesn't matter